src/masonite/routes/HTTPRoute.py

```diff
@@ -38,6 +38,9 @@ def __init__(
         self.controller_bindings = controller_bindings
         self.compile_route_to_regex()
 
+    def __str__(self):
```

Comment:
Really easier to debug to have a clear and unique reprensation of the object in console.